src/Microsoft.NET.Sdk.Razor/Sdk/Sdk.targets

```diff
@@ -11,11 +11,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0">
 
-  <PropertyGroup Condition="'$(RazorSdkTargetsPath)' == ''">
```

Comment:
The target in the package expects the Sdk to import this file. It affects a corner scenario where you only reference the package (but not the Sdk). This just leaves it up to Sdk\ package to import their own targets.

Reply:
It ended up being used in regular package references scenarios too, not just test. We don't particularly use this anymore, so removing it.